bazel-mode.el

```diff
@@ -21,9 +21,20 @@
 ;;
 ;;; Code:
 
+(defconst bazel-mode-syntax-table
+  (let ((table (make-syntax-table)))
+	; single and multiple line comment start
+	(modify-syntax-entry ?# "< 14" table)
+	; single line comment end
+	(modify-syntax-entry ?\n ">" table)
+	table)
+  "Syntax table for `bazel-mode'.")
+
 ;;;###autoload
 (define-derived-mode bazel-mode prog-mode "Bazel"
-  "Major mode for editing Bazel BUILD and WORKSPACE files.")
+  "Major mode for editing Bazel BUILD and WORKSPACE files."
+  :syntax-table bazel-mode-syntax-table
+  (font-lock-ensure))
```

Comment:
This also shouldn't be necessary.

Reply:
Same comment as above. If I remove this, my local examples stop working. https://emacs.stackexchange.com/questions/238/how-do-i-create-a-simple-syntax-table-for-a-major-mode seems relevant. Using font-lock-fontify-buffer results in a compilation error that points to font-lock-ensure. Adding that gives the right behavior.